sql/core/src/main/scala/org/apache/spark/sql/execution/SQLExecution.scala

```diff
@@ -77,7 +78,9 @@ object SQLExecution {
         try {
           sc.listenerBus.post(SparkListenerSQLExecutionStart(
             executionId = executionId,
-            description = callSite.shortForm,
+            description = Option(sparkSession.sparkContext
+              .getLocalProperty(SparkContext.SPARK_JOB_DESCRIPTION))
```

Comment:
You know, we should also redact i it. 

Also cc @gengliangwang

Reply:
@gatorsmile any suggestions on how to do it.? how do we identify 'sensitive' part of SQL which we need to hide.?

Reply:
https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala#L166-L171 

Please check whether the strings in SparkContext.SPARK_JOB_DESCRIPTION have been redacted. 

Thanks!

Reply:
@gatorsmile i have updated the PR to handle redact of string in SparkContext.SPARK_JOB_DESCRIPTION